import-service/src/functions/products.js

```diff
@@ -0,0 +1,14 @@
+import { prepareResponse, withCatchError } from '../utils/response';
+import { getSignedUrlByKey } from '../utils/bucket';
+
+export const importProductsFile = withCatchError(async (event) => {
+  console.log(`Request for "importProductsFile" has been received with ${event.queryStringParameters}`);
+  const { name } = event.queryStringParameters || {};
+  if (!name) {
+    return prepareResponse(event, 400, 'Name is required as query parameter.');
```

Comment:
Just a general remark - From security point of view it's best not to disclose to client what is wrong. Better log the exact reason for internal view and send a vague Bad Request to client, so we don't provide hints to a possible hacker on what exactly is wrong. No need to fix it here, just a remark :)